src/Rector/Contrib/PhpParser/SetLineRector.php

```diff
@@ -29,10 +28,19 @@
      */
     private $identifierRenamer;
 
-    public function __construct(MethodCallAnalyzer $methodCallAnalyzer, IdentifierRenamer $IdentifierRenamer)
```

Comment:
@TomasVotruba Is there a Fixer for `$identifierRenamer` instead of `$IdentifierRenamer`?

Reply:
You can extend this one: https://github.com/Symplify/Symplify/blob/master/packages/CodingStandard/README.md#property-name-should-match-its-type-if-possible-wrench